pkg/controller/installation/validation.go

```diff
@@ -411,6 +412,10 @@ func validateCustomResource(instance *operatorv1.Installation) error {
 		}
 	}
 
+	if operatorv1.IsFIPSModeEnabled(instance.Spec.FIPSMode) && instance.Spec.Variant == operatorv1.TigeraSecureEnterprise {
```

Comment:
I only picked this line from https://github.com/tigera/operator/pull/3455 and removed FIPS related logstorage config.

FIPS has not been supported since Jan 2024 and has never been supported in this branch.